bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -254,20 +284,32 @@
 		<label>@text/channel-type.mielecloud.temperature_current.label</label>
 		<description>@text/channel-type.mielecloud.temperature_current.description</description>
 		<category>Number:Temperature</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state pattern="%.1f %unit%" readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="ventilation_power">
 		<item-type>String</item-type>
 		<label>@text/channel-type.mielecloud.ventilation_power.label</label>
 		<description>@text/channel-type.mielecloud.ventilation_power.description</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Power</tag>
```

Comment:
This should rather be `Ventilation` or `Level`, has nothing to do with power consumption.

Reply:
Or maybe 'AirFlow' ??